cypress/component/basic/options-spec.js

```diff
@@ -6,14 +6,17 @@ const template = `
     </div>
   `
 
-const data = {
-  message: 'Hello Vue!',
-}
-
 describe('Mount component', () => {
   // hmm, there are no more options to pass
 
-  const component = { template, data }
+  const component = {
+    template,
+    data() {
```

Comment:
so during the test, what if we check the `.data` property and if it is an object, automatically  make it into a function? Otherwise, this is a breaking change.